developer-guides/rest-api/channels/README.md

```diff
@@ -5,6 +5,7 @@ These methods apply to public channels only. Use `groups.*` methods for private
 | Url | Short Description | Details Page |
 | :--- | :--- | :--- |
 | `/api/v1/channels.addAll` | Adds all of the users on the server to a channel. | [Link](addall/) |
+| `/api/v1/channels.addLeader` | Gives the role of Leader for a user in the current channel. | [Link](addleader/) |
```

Comment:
Can you please add this link and descriptions in [this file](https://github.com/RocketChat/docs/blob/87f678816c31a832cfab4e8db6cc858eb56b1435/developer-guides/rest-api/README.md) ?

Reply:
I forgot this file 😅 done!